updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -6,7 +6,21 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To get an update experience similar to connected clusters, you can use the following procedures to install and configure the OpenShift Update Service in a disconnected environment.
+To get an update experience similar to connected clusters, you can use the following procedures to install and configure the OpenShift Update Service (OSUS) in a disconnected environment.
+
+The following steps outline the high-level workflow on how to update a cluster in a disconnected environment using OSUS:
+
+. Configure access to a secured registry.
+
+. Update the global cluster pull secret to access your mirror registry.
+
+. Install the OSUS operator.
+
+. Create a graph data container image for OSUS.
+
+. Install the OSUS application and configure your clusters to use the local OSUS.
```

Comment:
```suggestion
. Install the OSUS application and configure your clusters to use the local OSUS instance.
```
I feel like I am missing a word at the end to describe an instance of the OSUS. WDYT?

Reply:
OpenShift Update Service is technically complete by itself, but yeah it does sound weird when used as an acronym like above, and instance would be a good word to include. I'll just write it out fully here, though